client/flow_test.go

```diff
@@ -716,41 +906,19 @@ func TestResetStateFlow(t *testing.T) {
 		}
 	}
 
-	resetReq := httprequests.ResetStateForm{
-		NewStateDBDSN: "",
-		UseOffset:     false,
-		Messages:      []string{msgToIgnore},
-	}
-	resetReqBz, err := json.Marshal(resetReq)
-	if err != nil {
-		t.Fatalf("failed to marshal ResetStateRequest: %v\n", err)
-	}
-
-	for i := startingPort; i < startingPort+numNodes; i++ {
-		if _, err := http.Post(fmt.Sprintf("http://localhost:%d/resetState", i),
-			"application/json", bytes.NewReader(resetReqBz)); err != nil {
-			t.Fatalf("failed to send HTTP request to reset state: %v\n", err)
-		}
-	}
-
-	time.Sleep(15 * time.Second)
-
-	runCtx, runCancel := context.WithCancel(context.Background())
-	for _, n := range nodes {
-		go n.run(nil, runCtx)
+	log.Print("\n\n\nResetting nodes states\n\n\n")
+	if err := resetNodesStates(nodes, []string{msgToIgnore}, false); err != nil {
+		t.Fatalf("failed to reset nodes states: %v", err)
 	}
-
+	time.Sleep(10 * time.Second)
```

Comment:
we've had issues with 10 seconds here. what do you think about moving all 10s to a single constant?

Reply:
re this particular step and sleep duration: on reset node state, all operations taken from the history are processed almost immediately, bringing the nodes to up-to-date state in about a single second. the next operations are operations taken from get_operations and they are actually not needed -- those are duplicated operations of the ones just processed from the history, and they all fail/are skipped. so even the 10-seconds sleep here is an overkill, but it's suitable just to keep the logs in historical-ish order.

re all sleeps making constant: even though sleep durations are very different from each other, we need this difference for different steps take different time, e.g. dkg ceremony takes about 10-20 seconds, message signing takes 5-10 seconds, but in some cases (like in new tests I've made with selective get_operations) signing takes 2-3 seconds at most, etc. and making it constant of 20 seconds will dramatically increase the test time. let me take a look and maybe I can come up with an enum-like way of making the sleep durations constant